src/runtime/plugins/vuetify-client-hints.server.ts

```diff
@@ -11,20 +11,32 @@ import {
   defineNuxtPlugin,
   useCookie,
   useNuxtApp,
-  useRequestEvent,
```

Comment:
I think this is a mistake. This imports is still required.

Reply:
you're right, I guess we need to improve nuxt module template

thx, I'll release a new version later